src/components/modules/document/common/Upload/modules/Video/modules/Recorder/modules/VideoStream/VideoStream.view.vue

```diff
@@ -151,14 +186,21 @@ export default {
       stream.getTracks().forEach(track => track.stop());
     },
 
-    onPlayEnd() {
-      this.$emit('update:recorder-state', RECORDER_STATE.IDLE_FOR_PLAY);
+    async onPlayEnd() {
+      if (this.stream && !this.stream.active) {
+        this.$emit('update:recorder-state', RECORDER_STATE.INITIALIZING);
```

Comment:
not understand, why we need reinit stream after playend?

Reply:
this is only about play, not any record state.
stream will be opened, when it will be needed for record only, not play.
you can revert this block

Reply:
so you allow to user ability to record 1sec selfies?

Reply:
@kazurus is this bug critical or can be omitted?

Reply:
playback and recording is not the same.
for playback you no need `stream`

Reply:
then how you want to catch moment when permissions are revoked? I know only Permissions API, but still in draft.

Reply:
i don't needed it here. because this is playback only

Reply:
it means, that issue with 1sec review stay here
for me it's bug, but if for @endpass/frontend it's okay, I revoke these changes

Reply:
Will discuss tomorrow

Reply:
please, describe the case, how to reproduce it

Reply:
- open firefox
- try to upload selfie, allow camera
- start to record and immediately drop permission
- recording ends, you will be able to upload 1sec selfie on server
- if on 3 step don't start recording, you'll anyways got ability to `continue` after permissions will be revoked

Reply:
@ifiri add comment here why we need this fix

Reply:
@dudiq https://github.com/endpass/endpass-auth/pull/342#discussion_r461564127

Reply:
add comment in code, not here

Reply:
ah, okay

Reply:
done